Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ however, it has to be formatted properly to pass verification tests.
 - Support for [Game rotation vector](https://developer.android.com/reference/android/hardware/Sensor#TYPE_GAME_ROTATION_VECTOR) sensor on Android
 - Duplicate Input Action Items in the new Input Action Asset Editor with Ctrl+D (Windows) or Cmd+D (Mac)
 - Selection of InputActionReferences from project-wide actions on fields that are of type InputActionReference. Uses a new advanced object picker that allows better searching and filtering of actions.
+- Reset project wide Input Settings to default via a new Kebab-menu in Input System Project Settings.
```

Comment:
Is this an official menu name we are using?

Reply:
Kebab-menu is the standard 3 vertical dots type menu (as opposed to the hamburger-menu with 3 lines).

Reply:
Kebab 👍